lib/reverse_sentence.rb

```diff
@@ -1,6 +1,66 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) where n is the number of characters in the sentence. This is because it is essentially multiplying
+# the number of characters in each word by 1 --> for each word, reverse each character in that word. So, it ends up being (1 * number of characters in word 1) + (1 * number of characters in word 2) etc. for each word,
+# which simplifies to adding together the number of characters in each word, which is essentailly O(n)
+# Space complexity: O(1)
```

Comment:
Since you are creating an array of breaks between words, this is actually O(m) where m is the number of words.